Assets/MixedRealityToolkit/Utilities/Editor/PackageManifest/PackageManifestUpdater.cs

```diff
@@ -13,6 +13,13 @@ namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
     /// </summary>
     internal static class PackageManifestUpdater
     {
+        private static string MSBuildRegistryUrl = "https://pkgs.dev.azure.com/UnityDeveloperTools/MSBuildForUnity/_packaging/UnityDeveloperTools/npm/registry";
+        private static string MSBuildRegistryName = "MS Build for Unity";
+        private static string[] MSBuildRegistryScopes = new string[] { "com.microsoft" };
```

Comment:
Do we want to be (can we be?) a little more specific here for the MSBuildForUnity scope? `com.microsoft` is pretty large.